docs/content/gds/admin/configuration/gds.en.md

```diff
@@ -1,7 +1,159 @@
+|---------------------|----------|---------|------------------------------------------------------------------|
+| GDS_BACKUP_ENABLED  | bool     | false   | If true, enables the backup background process.                  |
+| GDS_BACKUP_INTERVAL | duration | 24h     | The interval between database backups                            |
+| GDS_BACKUP_STORAGE  | string   |         | The path on disk to store database backups (required if enabled) |
+| GDS_BACKUP_KEEP     | int      | 1       | The number of backups to keep before cleaning up old backups     |
+
+Backups should not be enabled when using the TrtlDB database store! If the backups are enabled then the storage path is required.
+
+### Google Secrets
+
+GDS uses Google Secret Manager to store certificates and PKCS12 passwords and other sensitive information to ensure that it is encrypted and secure. Access to Google Secret Manager is configured as follows:
+
+| EnvVar                         | Type   | Default | Description                                                                                          |
+|--------------------------------|--------|---------|------------------------------------------------------------------------------------------------------|
+| GOOGLE_APPLICATION_CREDENTIALS | string |         | Path to the JSON credentials for the Google Service Account that has access to Google Secret Manager |
+| GOOGLE_PROJECT_NAME            | string |         | Name of the Google Project for API access to Google Secret Manager                                   |
+| GDS_SECRETS_TESTING            | bool   | false   | If set to true, uses a local in-memory "secret store" for testing and development                    |
+
+Note that the `GOOGLE_APPLICATION_CREDENTIALS` and `GOOGLE_PROJECT_NAME` are required if `GDS_SECRETS_TESTING` is `false`.
+
+### Sentry
+
+The GDS uses [Sentry](https://sentry.io/) to assist with error monitoring and performance tracing. Configure GDS to use Sentry as follows:
+
+| EnvVar                       | Type    | Default     | Description                                                                                       |
+|------------------------------|---------|-------------|---------------------------------------------------------------------------------------------------|
+| GDS_SENTRY_DSN               | string  |             | The DSN for the Sentry project. If not set then Sentry is considered disabled.                    |
+| GDS_SENTRY_SERVER_NAME       | string  |             | Optional - a server name to tag Sentry events with.                                               |
+| GDS_SENTRY_ENVIRONMENT       | string  |             | The environment to report (e.g. development, staging, production). Required if Sentry is enabled. |
+| GDS_SENTRY_RELEASE           | string  | {{version}} | Specify the release of Ensign for Sentry tracking. By default this will be the package version.   |
```

Comment:
Should `Ensign` be in the description?

Reply:
Ooops - good catch! I'll fix it in the next PR.